### DISCUSSION TITLE
Unable to delete organizations

### DISCUSSION THREAD

**Discussion by tobiasnix** [community/community#22965] [New to GitHub]

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/tobiasnix">@tobiasnix</a>,</p>
<p>Thank you for being here! Please check your email for a message from private support. </p>

**Reply by anuteshome** [reply]
ok

**Comment by tobiasnix** [comment]
<p>Thanks, my issue was solved. Is it possible to delete this whole thread pls?</p>

**Comment by aymansalkhatib** [comment]
I also cannot delete an organization from my account due to blocking restrictions. What is the solution?

**Comment by LucaArgentieri** [comment]
I cannot delete an organization because the overview page and settings give me a 404 error

**Comment by Rasket** [comment]
I cannot delete an organization. Page says i need to contact support

**Reply by MarkusStefan** [reply]
Same issue for me!

**Comment by MarkusStefan** [comment]
I cannot delete an organization I have created. In the danger section, it says i need to contact support.

**Comment by Strafato** [comment]
Same here

**Comment by Wolf2000Pi** [comment]
I cannot delete an organization I have created. In the danger section, it says i need to contact support.

**Reply by MarkusStefan** [reply]
The issue is known and most likely a misclassification by their spam-detector.
Just do so and raise a support ticket. They'll reach out to you, eventually. It took 2 months for them to get back to me and unflag my organization.

**Reply by Rasket** [reply]
I simply created second account, add it to organization and remove my self

**Comment by ghost** [comment]
我也无法删除我的组织

**Comment by KillerX629** [comment]
I can't delete my organization neither
